.gitignore

```diff
@@ -94,3 +91,5 @@ CMakeLists.txt.user
 
 node_modules
 package-lock.json
+
+snapshots
```

Comment:
when is this one a problem?

Reply:
Not sure where these come from, I just saw these show up in my "Untracked" files:

```
~/github/enf/leap gitignore_cleanup:main ⇡5 !1 ?14 ❯ ls snapshots plugins/snapshots                                                                                                             
plugins/snapshots:
snapshot-000000062d3e86cb288ecdfd37e4ba5f313b97c04b3c8f47e010275c4a37fe62.bin

snapshots:
snapshot-000000062d3e86cb288ecdfd37e4ba5f313b97c04b3c8f47e010275c4a37fe62.bin  snapshot-00000006c38bac7f934ac5cf688d8530bac0d112e0404548e391aef2b404bda2.bin
```